docs/services/slam/cartographer/_index.md

```diff
@@ -34,318 +22,467 @@ The source code for this module is available on the [`viam-cartographer` GitHub
+           },
+           "camera": {
+             "name": "rplidar",
+             "data_frequency_hz": "5"
+           },
+           "enable_mapping": true,
+           "use_cloud_slam": true,
+           "existing_map": "${packages.slam_map.test-map-1}/internalState.pbstream"
+         },
+         "name": "slam",
+         "type": "slam",
+         "namespace": "rdk",
+         "model": "viam:slam:cartographer"
+       },
+       {
+         "name": "Data-Management-Service",
+         "type": "data_manager",
+         "attributes": {
+           "tags": [],
+           "additional_sync_paths": [],
+           "sync_interval_mins": 0.1,
+           "capture_dir": ""
+         }
+       }
+     ],
+     "components": [
+       {
+         "namespace": "rdk",
+         "attributes": {},
+         "depends_on": [],
+         "service_configs": [
+           {
+             "attributes": {
+               "capture_methods": [
+                 {
+                   "disabled": false,
+                   "method": "NextPointCloud",
+                   "capture_frequency_hz": 5
+                 }
+               ]
+             },
+             "type": "data_manager"
+           }
+         ],
+         "name": "rplidar",
+         "model": "viam:lidar:rplidar",
+         "type": "camera"
+       }
+     ],
+     "packages": [
+       {
+         "name": "test-map-1",
+         "version": "1697208847",
+         "package": "d1c224e8-483e-4cc7-980f-76b89d8fb507/test-map-1",
+         "type": "slam_map"
+       }
+     ]
+   }
+   ```
+
+   {{% /tab %}}
+   {{< /tabs >}}
+
+   {{< alert title="Tip" color="tip" >}}
+   Be aware that data is not only captured when a slam session is running.
+   Data capture and the resulting charges continue, unless you [turn off data capture for your sensors](/services/data/configure-data-capture/).
+   {{< /alert >}}
+
+   For more information about the configuration attributes, see [Attributes](#attributes).
+
+2. Start a mapping session
+
+   Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
+   On the cartographer panel, you can start a mapping session.
+
+   When you start a mapping session:
+
+   - Viam spins up an instance of the `cartographer` module in the cloud to execute the Cartographer algorithm.
+   - Viam's [Data Capture](/services/data/configure-data-capture/) continuously monitors and captures your robot's sensor data while the robot is running.
+   - Cartographer uses the data captured from when you click **Start session** until you click **End session** to create the new map based on the existing map.
+     Once you click **End session**, the map is uploaded to the cloud and visible on your **Location** page under **SLAM library**.
+
+   Enter a name for your new map and click **Start session**.
+   Wait for the slam session to finish starting up in the cloud, which **takes about 2 minutes**.
```

Comment:
This is incorrect: you don't enter a map name since you're updating a map

Also, this ends a little abruptly without saying that the rest is similar to "Create a new map" mode, so perhaps we could say that here.